pom.xml

```diff
@@ -218,6 +220,14 @@
                                 <goal>shade</goal>
                             </goals>
                             <configuration>
+                                <transformers>
+                                    <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+                                        <manifestEntries>
+                                            <!-- Add `ecocode-rules-specifications` version in plugin metadata -->
+                                            <X-EcoCode-Rules-Specifications>${project.version}</X-EcoCode-Rules-Specifications>
```

Comment:
Why do you need the version of rules specification in the plugin manifest?

Reply:
just for traceability